src/Components/Carousel/Carousel.css

```diff
@@ -3,15 +3,44 @@
   display: flex;
   width: 100%;
   height: 100%;
+  align-content: center;
 }
 
 .main-img{
   width: 100%;
-  max-width: 750px;
-  max-height: 950px;
+  max-width: 400px;
```

Comment:
The unit needs to be relative. Some tablets might have a hdpi display.

Reply:
the unit is relative but i need to put a max width and height that is fix or else the picture take all the screen